### PR TITLE
Cb1276 snowtify

### PR DESCRIPTION
**Pull request by Anilreddy999**

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -140,6 +157,42 @@ pipeline:
...
+  complete-snow-change-dev:
+    image: quay.io/ukhomeofficedigital/snowtify:latest
+    secrets:
+      - source: DEV_SERVICE_NOW_PASSWORD
+        target: SNOW_TEST_PASS
+      - source: DEV_SERVICE_NOW_USERNAME
+        target: SNOW_TEST_USER
+    environment:
+      - SNOW_DESC=${DRONE_COMMIT_MESSAGE}
+      - SNOW_ENDPOINT=https://cto-change-dev.internal.cto-notprod.homeoffice.gov.uk/create
```
Comment (belindac-vd) [inline .drone.yml]: This needs to be       - SNOW_ENDPOINT=https://cto-change-dev.internal.cto-notprod.homeoffice.gov.uk/update

File: .drone.yml [hunk 2]
```diff
@@ -140,6 +157,42 @@ pipeline:
...
+  cancel-snow-change-dev:
+    image: quay.io/ukhomeofficedigital/snowtify:latest
+    secrets:
+      - source: DEV_SERVICE_NOW_PASSWORD
+        target: SNOW_TEST_PASS
+      - source: DEV_SERVICE_NOW_USERNAME
+        target: SNOW_TEST_USER
+    environment:
+      - SNOW_DESC=${DRONE_COMMIT_MESSAGE}
+      - SNOW_ENDPOINT=https://cto-change-dev.internal.cto-notprod.homeoffice.gov.uk/create
```
Comment (belindac-vd) [inline .drone.yml]: This needs to be       - SNOW_ENDPOINT=https://cto-change-dev.internal.cto-notprod.homeoffice.gov.uk/update

File: .drone.yml (hunk 1 above)
Comment (Anilreddy999) [reply .drone.yml]: Sorry, corrected now. Please review

File: .drone.yml (hunk 2 above)
Comment (Anilreddy999) [reply .drone.yml]: Sorry, corrected now. Please review

**Pull request merged**